SourceX/DiabloUI/diabloui.cpp

```diff
@@ -266,6 +266,8 @@ bool UiFocusNavigation(SDL_Event *event)
 		case SDLK_DELETE:
 			UiFocusNavigationYesNo();
 			return true;
+		default:

```

Comment:
Does you editor automatically add default cases?

Reply:
This is to avoid the "unhandled cases in switch warning".

Basically the idea is that if you have a `default` case, it is intended that you didn't handle all the enum elements.

If you don't have a default case, the compiler expects you to handle all the cases and warns you if you forgot one. This is super useful for things like tagged unions.

Reply:
+1